glide.lock

```diff
@@ -341,7 +341,7 @@ imports:
   version: 1560c1005499d61b80f865c04d39ca7505bf7f0b
 - name: github.com/spf13/viper
   version: c975dc1b4eacf4ec7fdbf0873638de5d090ba323
-  repo: https://github.com/jimmidyson/viper.git
+  repo: https://github.com/spf13/viper.git
```

Comment:
@hferentschik Oh ya, I tested with removing glide.lock. but found that `spf13/viper.git` version is same as ` c975dc1b4eacf4ec7fdbf0873638de5d090ba323`.